pyproject.toml

```diff
@@ -60,8 +64,12 @@ test-skip = [
     "*-*linux_i686",
 ]
 
+[tool.cibuildwheel.linux]
+archs = ["auto", "aarch64"]
+
 [tool.cibuildwheel.macos]
 archs = ["x86_64", "universal2", "arm64"]
+environment = { MACOSX_DEPLOYMENT_TARGET = "10.15" }
```

Comment:
We don't need this. `setup.py` will set this env variable when needed, which ensures the build process to work without using cibuildwheel.

Reply:
Do we need it in the workflow then?

Reply:
I think we don't need to explicitly specify it in the workflow

Reply:
Looks like we do: https://github.com/y-scope/clp-ffi-py/actions/runs/8307258601/job/22736272998.

Do you have a solution or should we add it back to `pyproject.toml`?

Reply:
Sorry my bad. Here's the solution: https://github.com/y-scope/clp-ffi-py/pull/52